geode-core/src/distributedTest/java/org/apache/geode/security/AuthExpirationDUnitTest.java

```diff
@@ -66,6 +67,12 @@
 
   private ClientVM clientVM;
 
+  @Before
```

Comment:
It looks like you made some junit5 changes elsewhere. Might be good to do it here.